src/components/Book.jsx

```diff
@@ -15,21 +15,48 @@ function Book(props) {
   };
 
   return (
-    <div>
-      <ul>
-        <li>
-          {title}
-          {' '}
-          by
-          {author}
-          {' '}
-          {category}
-&nbsp;
-          <button onClick={() => handleDel()} type="button">
-            Delete
+    <div className="book">
+      <div className="p-1">
+        <div className="titles">
+          <h4 className="category">{category}</h4>
+          <h2 className="title">{title}</h2>
+          <p className="author">{author}</p>
+        </div>
+        <div className="buttons">
+          <button className="btn comment-btn" type="submit">
+            Comment
           </button>
-        </li>
-      </ul>
+          <button
+            onClick={() => handleDel()}
+            type="button"
+            className="btn remove-btn"
+          >
+            Remove
+          </button>
+          <button className="btn edit-btn" type="submit">
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="p-2">
+        <div className="circle">
+          <svg>
+            <circle cx="20" cy="20" r="20" />
+            <circle cx="20" cy="20" r="20" />
+          </svg>
+        </div>
+        <div className="infos">
+          <h3>8%</h3>
+          <span>completed</span>
+        </div>
+      </div>
+      <div className="p-3">
+        <p className="chpt">Current chapter</p>
+        <p className="intro">Introduction</p>
+        <button className="btn-update" type="submit">
+          Update process
+        </button>
+      </div>
```

Comment:
- [ ] Please stick to the [Zaplin](https://zeplin.io/) design you missed `border-left` so kindly add `border-left` to match your design to the [Zaplin](https://zeplin.io/) design :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-23 16-12-25](https://github.com/husush23/Book-Store/assets/45226848/f912dbc9-dee2-4755-8d5f-ed7e05f5c491)

**Figma Design** 
![Screenshot from 2023-06-23 16-12-18](https://github.com/husush23/Book-Store/assets/45226848/70d1e432-8704-4d24-804b-eda4a1c9173d)

- [ ] Please `uppercase` all the letters of a button and also the `Current chapter` text to match your design with [Zaplin](https://zeplin.io/) design :+1: 
- [ ] Please change color of the button text it's not the same as the [Zaplin](https://zeplin.io/) design :+1: